en/sql/schema/view_stmt.rst

```diff
@@ -294,15 +294,55 @@ You can drop a query defined in the query specification by using the **DROP QUER
 COMMENT Clause
 --------------
 
-You can change a view's comment with **COMMENT** clause of **ALTER VIEW** syntax.
+You can change a view's comment and column comments with **COMMENT** clause of **ALTER VIEW** syntax.
 
 ::
 
-    ALTER [VIEW | VCLASS] view_name COMMENT [=] 'view_comment';
+    ALTER [VIEW | VCLASS] view_name
+    COMMENT [=] 'view_comment_string';
+    COMMENT ON {COLUMN | CLASS ATTRIBUTE} <column_comment_definition> [, <column_comment_definition>] ;
+
+        <column_comment_definition> ::= column_name [=] 'column_comment_string'
+
+*   *view_name*: specifies the name of a view to be modified.
+*   *column_name*: specifies the name of a column to be modified.
+*   *view_comment_string*: specifies a view's comment.
+*   *column_comment_string*: specifies a column's comment.
+
+The following example shows how to change a view's comments.
+
+.. code-block:: sql
+
+    ALTER VIEW v1 COMMENT = 'changed view v1 comment';
+
+You can change the column comment by specifying one or more columns after the ON COLUMN keyword.
+The following example shows how to change a column's comments.
 
 .. code-block:: sql
 
-    ALTER VIEW b_view COMMENT = 'changed view comment';
+    ALTER VIEW v1 COMMENT ON COLUMN c1 = 'changed view column c1 comment';
+    ALTER VIEW v1 COMMENT ON COLUMN c2 = 'changed view column c2 comment', c3 = 'changed view column c3 comment';
+
+The below is a syntax to show a column's comment.
```

Comment:
The below is a syntax to show a column's comment. -->  Below is a syntax to show a column's comment.  (Delete 'The')

Reply:
Thank you. Fixed in '1989cf0' commit.